php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -768,6 +771,9 @@ private boolean isOnlyInheritdoc(PHPDocBlock phpDocBlock) {
         }
 
         static String replaceInlineInheritdoc(String description, String inheritdoc) {
```

Comment:
OT: Again, missing annotations for `null` values.

Reply:
Resolving because of no change of this line in this MR (but still, would be nice to have it done).